### PR TITLE
Added help command

### PR DESCRIPTION
**Pull request by Nurdoidz**
For later:

- pagination as we add more commands

### CODE REVIEW

File: Commands/Select.java [hunk 1]
```diff
@@ -1,55 +1,86 @@
...
-        return arguments.length >= minArguments;
+    public String getBasicUsage() {
+        return "select [unit]";
     }
 
     @Override
-    public boolean hasTooManyArguments(String[] arguments) {
-        return arguments.length > maxArguments;
+    public String getDescription() {
+        return "Asks a (selected) unit to attack another.";
```
Comment (MrGeektron) [inline Commands/Select.java]: Is this supposed to be the same as attack command?

**Review by MrGeektron — CHANGES_REQUESTED** [review]
Everything looks good except for one definition. Also is select supposed to crash with no arguments?

File: Commands/Select.java (hunk 1 above)
Comment (Nurdoidz) [reply Commands/Select.java]: No... I think the select command was masquerading as another command to get more attention...

**Comment by Nurdoidz** [comment]
The **select** and **move** commands now have basic usage and do not crash the game. I will work on writing tests for all the code I wrote later, but this should get you going on handling the complexities of those commands.

**Review by MrGeektron — APPROVED** [review]

**Pull request merged**